helm/hades/templates/hades-operator-role.yaml

```
@@ -0,0 +1,47 @@
+  name: {{ .Values.rbac.hadesOperator.role.name }}
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps","events","pods","pods/log","secrets"]
+    verbs: ["get","list","watch","create","update","patch","delete"]
+
+  - apiGroups: ["batch"]
+    resources: ["jobs"]
+    verbs: ["get","list","watch","create","update","patch","delete"]
+
+  - apiGroups: ["build.hades.tum.de"]
+    resources: ["buildjobs","buildjobs/status","buildjobs/finalizers"]
```

YAMLlint (1.37.1)

[warning] too few spaces after comma

(commas)